Common/Orders/OrderTypes.cs

```diff
@@ -154,4 +154,91 @@ public enum OrderStatus {
         CancelPending = 8
     }
 
+    /// <summary>
+    /// Interactive Brokers algorithm selection
+    /// </summary>
+    public enum OrderAlgorithm
```

Comment:
These are specific to interactive brokers and should be under the `QuantConnect.Brokerages.InteractiveBrokers` namespace -- still located in the `Common` project -- but we may want a more generalized way of handling brokerage specific order parameters rather than having to define special types for every brokerage -- so not sure how I feel about this yet